src/Yield.Infrastructure/Repositories/Interfaces/IPlotRepository.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Yield.Core.Entities;
+
+namespace Yield.Infrastructure.Repositories.Interfaces
+{
+    public interface IPlotRepository
+    {
+        Task<IEnumerable<Plot>> GetPlots(Guid AllotmentId);
```

Comment:
Just seen this one as well. ;)
```suggestion
        Task<IEnumerable<Plot>> GetPlots(Guid allotmentId);
```